openmrs/apps/clinical/formConditions.js

```diff
@@ -382,5 +382,1566 @@ Bahmni.ConceptSet.FormConditions.rules = {
+		} else {	
+			conditions.hide.push("Protein (CSF) Test Date");
+		}	
+		return conditions;	
+	},
+	'Glucose (CSF)': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Glucose (CSF)'];
+		if (value && value !== "") {	
+			conditions.show.push("Glucose (CSF) Test Date");
+		} else {	
+			conditions.hide.push("Glucose (CSF) Test Date");
+		}	
+		return conditions;	
+	},
+	'Chloride (CSF)': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Chloride (CSF)'];
+		if (value && value !== "") {	
+			conditions.show.push("Chloride (CSF) Test Date");
+		} else {	
+			conditions.hide.push("Chloride (CSF) Test Date");
+		}	
+		return conditions;	
+	},	
+	'Urine dipstick': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Urine dipstick'];
+		if (value && value !== "") {	
+			conditions.show.push("Urine dipstick Test Date");
+		} else {	
+			conditions.hide.push("Urine dipstick Test Date");
+		}	
+		return conditions;	
+	},
+	'Hemoglobin electrophoresis': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Hemoglobin electrophoresis'];
+		if (value && value !== "") {	
+			conditions.show.push("Hemoglobin electrophoresis Test Date");
+		} else {	
+			conditions.hide.push("Hemoglobin electrophoresis Test Date");
+		}	
+		return conditions;	
+	},
+	'Protein electrophoresis': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Protein electrophoresis'];
+		if (value && value !== "") {	
+			conditions.show.push("Protein electrophoresis Test Date");
+		} else {	
+			conditions.hide.push("Protein electrophoresis Test Date");
+		}	
+		return conditions;	
+	},
+	'HbA1C': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['HbA1C'];
+		if (value && value !== "") {	
+			conditions.show.push("HbA1C Test Date");
+		} else {	
+			conditions.hide.push("HbA1C Test Date");
+		}	
+		return conditions;	
+	},
+	'Iron level': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Iron level'];
+		if (value && value !== "") {	
+			conditions.show.push("Iron level Test Date");
+		} else {	
+			conditions.hide.push("Iron level Test Date");
+		}	
+		return conditions;	
+	},
+	'Total cholesterol': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['Total cholesterol'];
+		if (value && value !== "") {	
+			conditions.show.push("Total cholesterol Test Date");
+		} else {	
+			conditions.hide.push("Total cholesterol Test Date");
+		}	
+		return conditions;	
+	},
+	'HDL': function(formName, formFieldValues) {
+		var conditions = {show: [], hide: []};	
+		var value = formFieldValues['HDL '];
```

Comment:
@lauravignoli please remove the space after `HDL`

Reply:
Fixed